projects/storefrontlib/src/cms-components/cart/cart-coupon/applied-coupons/applied-coupons.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Voucher, CartVoucherService } from '@spartacus/core';
+import { ICON_TYPE } from '../../../../cms-components/misc/icon/index';
+
+@Component({
+  selector: 'cx-applied-coupons',
+  templateUrl: './applied-coupons.component.html',
```

Comment:
can we set `changeDetection: ChangeDetectionStrategy.OnPush,` here?

Reply:
Done